volaparrot/commands/upload.py

```diff
@@ -183,6 +188,29 @@ def cmd_unlockfile(self, remainder, msg):
         self.unlock_file(remainder)
         return True
 
+class FilesUploadCommand(UploadDownloadCommand, Command):
```

Comment:
I know you just copied phrases stuff, but no need for a new command here, just add it in UploadDownloadCommand